sel_phase_linac.py

```diff
@@ -106,11 +106,12 @@ def straighten_cheeto(self) -> bool:
             elif startVal + step > 180:
                 step = step - 360
             
-            print(f"{prefix}{self}{suffix}")
-            print("\t", "old:", f"{startVal:7.2f}")
-            print("\t", "new:", f"{startVal + step:7.2f}")
-            print("\t", "step:", f"{step:5.2f}")
-            print("\t", "chi^2", f"{chisum:.2g}\n")
+#            print(f"{prefix}{self}{suffix}")
+#            print("\t", "old:", f"{startVal:7.2f}")
+#            print("\t", "new:", f"{startVal + step:7.2f}")
+#            print("\t", "step:", f"{step:5.2f}")
+#            print("\t", "chi^2", f"{chisum:.2g}\n")
+            print(f"{prefix}{self}{suffix}  old: {startVal:7.2f} new: {startVal+step:7.2f} step: {step:5.2f} chi^2: {chisum:.2g}")
```

Comment:
Oh man I find this really hard to read though 😂 Can we shorten it and only have the step? Are the other things necessary?